content/sifixes/src/header/si/si_shapes.uc

```diff
@@ -562,15 +562,12 @@ enum npc_shapes
 	SHAPE_NAKED_BLONDE_WOMAN = 930,
 	SHAPE_NAKED_BLACK_WOMAN = 933,
 	SHAPE_NAKED_BROWN_WOMAN = 938,
-	SHAPE_AUTOMATON_WITH_CAPE = 945,		// Isstanar
+	SHAPE_AUTOMATON_WITH_CAPE = 945,			// Isstanar
 	SHAPE_HEALER = 946,
 	SHAPE_OPHIDIAN_SOLDIER = 947,
 	SHAPE_CATMAN = 957,
 	SHAPE_FEMALE_AVATAR = 989,				// Avatar from BG per prior notes
-	SHAPE_FEMALE_MAGE_IN_BLUE_GREEN = 1015,	// Elissa
-	
-	SHAPE_KNIGHT_OF_MONITOR = 1038,			// Caladin custom artwork
-	SHAPE_PIKEMAN_DARK = 1048				// Johnson custom artwork
```

Comment:
They're not part of the original game, but rather artwork I made. I have not released them as SHP files yet, so specifying them here does no good.